### ISSUE TITLE
Use TimeControlNode for Play/Pause/Step buttons

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/collision-lab#43]
From phetsims/scenery-phet#561. A scenery-phet/TimeControlNode was created that combines the PlayPauseButton and StepForward button and includes PhET-iO and PDOM instrumentation. It should be used in this sim. Assigning to responsible dev.

**Comment by veillette** [comment]
Collision lab has a Play/Pause button and a stepForward button. However, it also has a stepBackward Button. 
![image](https://user-images.githubusercontent.com/7633344/70754080-47ce5280-1d04-11ea-8b49-cc9157b66a6e.png)

I believe that there are other simulations with a stepBackward Button.
 
@jessegreenberg, given this, should I go ahead and use the TimeControlNode.

**Comment by jessegreenberg** [comment]
Thanks @veillette, I think it would be good to add the StepBackwardButton to TimeControlNode. Ill do that and then reassign.

**Comment by jessegreenberg** [comment]
@veillette there is now an option to include a StepBackwardButton with TimeControlNode, is that sufficient for usage in thsi sim?

**Comment by veillette** [comment]
Thanks for your prompt response @jessegreenberg , the StepBackwardButton is all that was needed for the simulation.

**Comment by brandonLi8** [comment]
In the commit above,`TimeControlBox` (sim-specific solution) was removed and was replaced with `SCENERY_PHET/TimeControlNode`. @veillette please review and close.

**Comment by veillette** [comment]
Thanks @brandonLi8 . 
I think we should be able to take it even further by incorporating the normal/slow radio buttons within the timeControlNode by providing it with an optional Property (isSlowMotionProperty). 
On the model side, we would need to change the model slightly, by getting rid of the NumberProperty SpeedProperty, in favor of a BooleanProperty IsSlowMotionProperty.  

Can you look into it @brandonLi8 ?

**Comment by veillette** [comment]
@jessegreenberg, it would be useful to add some additional methods for laying out this node.
Collision lab assumes that the Play/Pause Button is horizontally centered with the middle of the playgrid. However with the addition of the slow/normal radioButtons, the Play/Pause Button is no longer the center of the node.   
Can we expose the XCenter of the play pause button with respect to the left via a public method?

**Comment by veillette** [comment]
@brandonLi8 , I went ahead and added the slow/normal buttons to the TimeControlNode. In the process I deleted the speedRadioButton since there are now superfluous. The model has now a isSlowMotionProperty that can be leveraged in the TimeControlNode. 

The centering of the PlayPause with the grid is a bit tricky and I merely added a magic number to center it. This may be resolved by https://github.com/phetsims/scenery-phet/issues/564.
